index.js

```diff
@@ -31,18 +33,32 @@ class APIClient {
 
         const { extensionId } = this;
 
-        return got.put(uploadExistingURI(extensionId), {
-            headers: this._headers(await token),
-            body: readStream,
-        }).json();
+        return got
+            .put(uploadExistingURI(extensionId), {
+                headers: this._headers(await token),
+                body: readStream,
+            })
+            .json();
     }
 
     async publish(target = 'default', token = this.fetchToken()) {
         const { extensionId } = this;
 
-        return got.post(publishURI(extensionId, target), {
-            headers: this._headers(await token),
-        }).json();
+        return got
+            .post(publishURI(extensionId, target), {
+                headers: this._headers(await token),
+            })
+            .json();
+    }
+
+    async get(projection = 'DRAFT', token = this.fetchToken()) {
```

Comment:
Since they're both optional, this should be an object:
```suggestion
    async get({projection = 'DRAFT', token = this.fetchToken()}) {
```

so it can be called with `get({token: 'kdjfnglaskjgnlsdf'})` or `get({projection: 'DRAFT'})`

Reply:
But `async publish() {}` also had 2 optional params, dont we wanna stay consistent ?

Reply:
Good point. Should be done all at once at some point later